### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by thimslugga**
Added Templates as it caused the doc example to return an error.

### CODE REVIEW

File: apistar/__init__.py [hunk 1]
```diff
@@ -8,9 +8,11 @@
 from apistar.app import App
 from apistar.http import Request, Response
 from apistar.routing import Include, Route
+from apistar.templating import Templates
 from apistar.test import TestClient
 
 __version__ = '0.1.17'
 __all__ = [
-    'App', 'Include', 'Route', 'Request', 'Response', 'TestClient'
+    'App', 'Include', 'Route', 'Request', 'Response', 'Templates,
```
Comment (tomchristie) [inline apistar/__init__.py]: Missing a closing `'` character.

Comment (thimslugga) [inline apistar/__init__.py]: whoops, my bad!

File: apistar/__init__.py [hunk 2]
```diff
@@ -8,9 +8,11 @@
 from apistar.app import App
 from apistar.http import Request, Response
 from apistar.routing import Include, Route
+from apistar.templating import Templates
 from apistar.test import TestClient
 
 __version__ = '0.1.17'
 __all__ = [
-    'App', 'Include', 'Route', 'Request', 'Response', 'TestClient'
+    'App', 'Include', 'Route', 'Request', 'Response', 'Templates',
```
Comment (tomchristie) [inline apistar/__init__.py]: Trailing whitespace :)

Comment (thimslugga) [inline apistar/__init__.py]: thanks for pointing that out, should be fixed :)

**Comment by tomchristie** [comment]
Thanks for this! 😎

**Pull request merged**